.vscode/settings.json

```diff
@@ -85,6 +85,83 @@
     "C_Cpp.formatting": "disabled",
     "astyle.astylerc": "${workspaceRoot}/.astylerc",
     "files.associations": {
-        "vector": "cpp"
+        "vector": "cpp",
```

Comment:
Are the changes in this file intended to be in this PR?

Reply:
Hmm no. 
I thought this file was in the .gitignore before?